src/main/java/com/qaprosoft/carina/demo/gui/components/Header.java

```diff
@@ -69,4 +69,10 @@ public void verifyHeaderBasicElements(SoftAssert softAssert) {
         softAssert.assertTrue(logInIcon.isElementPresent(), "Log in button was not found!");
         softAssert.assertTrue(signUpButton.isElementPresent(), "Sign up button was not found!");
     }
+
+    public DropDownSearchForm openDropDownSearchForm(String text) {
+        searchField.type(text);
+        searchField.click();
```

Comment:
Why you click one more time ?

Reply:
Because Search form won't open if I don't click on search field.